lib/i18n/sfDateFormat.class.php

```diff
@@ -229,7 +229,7 @@ public function format($time, $pattern = 'F', $inputPattern = null, $charset = '
     for ($i = 0, $max = count($tokens); $i < $max; $i++)
     {
       $pattern = $tokens[$i];
-      if ($pattern[0] == "'" && $pattern{strlen($pattern) - 1} == "'")
+      if ($pattern[0] == "'" && $pattern[(strlen($pattern) - 1)] == "'")
```

Comment:
Very minor thing, but could the inner parentheses be skipped here? They are not needed and don't improve readability.